src/blocks/common.blocks/post/_view/post_view_full.bemtree

```diff
@@ -123,7 +123,9 @@ block('post').mod('view', 'full')(
     }),
 
     mode('issue')(function () {
-        var repo = this.ctx.content;
+        var repo = this.ctx.content,
+            // Match on the tag /technology/bemhtml/[2.3.0]/reference/
+            isTag = /.*\/v?\d\.\d\.\d\/.*/.test(this.data.node.url);
```

Comment:
@tadatuta 
1. I dont know :) update PR
2. Well, we expect that all our libs will be semver-named. Otherwise, we shoul make this verification on server.